assets/scripts/wplf-form.js

```diff
@@ -26,6 +26,7 @@
 
       fetch(ajax_object.ajax_url  + '?action=wplf_submit', {
         method: "POST",
+        credentials: ajax_object.ajax_credentials || 'omit', // if it's not defined, omit. 
```

Comment:
you already broke current behaviour when you switched from $.ajax to window.fetch :)

I don't see why it would ever be intentional to omit cookies from form submissions.